### PR TITLE
feat(crons): Bulk edit frontend modal

### PR DESCRIPTION
**Pull request by davidenwang**
image

dependent on: https://github.com/getsentry/sentry/pull/63525

https://www.figma.com/file/cfcxnhwGet0cHZKEZXY5kV/Crons?type=design&node-id=2709-25029&mode=design&t=inZDh33mug00rcoc-0

### CODE REVIEW

File: static/app/components/modals/bulkEditMonitorsModal.tsx [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+  const {
+    data: monitorList,
+    getResponseHeader: monitorListHeaders,
+    isLoading,
+  } = useApiQuery<Monitor[]>(queryKey, {
+    staleTime: 0,
+  });
+  const monitorPageLinks = monitorListHeaders?.('Link');
+
+  const headers = [t('Monitor'), t('State'), t('Muted'), t('Schedule')];
```
Comment (davidenwang) [inline static/app/components/modals/bulkEditMonitorsModal.tsx:86]: PR was getting long, will add sorts + header-level checkbox in follow-up PR

File: static/app/components/modals/bulkEditMonitorsModal.tsx [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+                    <Checkbox
+                      checked={isMonitorSelected(monitor)}
+                      onChange={() => {
+                        handleToggleMonitor(monitor);
+                      }}
+                    />
+                    <Text>{monitor.slug}</Text>
+                  </MonitorSlug>
+                  <Text>{monitor.status === 'active' ? t('Active') : t('Disabled')}</Text>
+                  <Text>{monitor.isMuted ? t('Yes') : t('No')}</Text>
```
Comment (evanpurkhiser) [inline static/app/components/modals/bulkEditMonitorsModal.tsx:157]: Should we use a `Tag` component here? red / green. Might make it easier to see

Comment (evanpurkhiser) [reply static/app/components/modals/bulkEditMonitorsModal.tsx:157]: cc @vuluongj20

**Review by evanpurkhiser — APPROVED** [review]
Code looks pretty good

**Pull request merged**